File: API/Interface/Models/LoanApplication.cs
using System;

namespace JestersCreditUnion.Interface.Models
{
    public class LoanApplication
    {
        public Guid? LoanApplicationId { get; set; }
        public short? Status { get; set; }
        public string StatusDescription { get; set; }
        public DateTime? ApplicationDate { get; set; }
        public string BorrowerName { get; set; }
        public DateTime? BorrowerBirthDate { get; set; }
        public Address BorrowerAddress { get; set; }
        public string BorrowerEmailAddress { get; set; }
        public string BorrowerPhone { get; set; }
        public string BorrowerEmployerName { get; set; }
        public DateTime? BorrowerEmploymentHireDate { get; set; }
        public decimal? BorrowerIncome { get; set; }
        public string CoBorrowerName { get; set; }
        public DateTime? CoBorrowerBirthDate { get; set; }
        public Address CoBorrowerAddress { get; set; }
        public string CoBorrowerEmailAddress { get; set; }
        public string CoBorrowerPhone { get; set; }
        public string CoBorrowerEmployerName { get; set; }
        public DateTime? CoBorrowerEmploymentHireDate { get; set; }
        public decimal? CoBorrowerIncome { get; set; }
        public decimal? Amount { get; set; }
        public string Purpose { get; set; }
        public decimal? MortgagePayment { get; set; }
        public decimal? RentPayment { get; set; }
        public DateTime? CreateTimestamp { get; set; }
        public DateTime? UpdateTimestamp { get; set; }
    }
}

File: Client/Member/src/app/models/loan-application.ts
import { Address } from './address';
import { LoanApplicationComment } from './loan-application-comment';
export class LoanApplication {
    StatusDescription: string | null = null;
    LoanApplicationId: string | null = null;
    BorrowerName: string = "";
    BorrowerBirthDate: string | null = null;
    BorrowerAddress: Address | null = null;
    BorrowerEmailAddress: string = "";
    BorrowerPhone: string = "";
    BorrowerEmployerName: string = "";
    BorrowerEmploymentHireDate: string | null = null;
    BorrowerIncome: number | null = null;
    CoBorrowerName: string = "";
    CoBorrowerBirthDate: string | null = null;
    CoBorrowerAddress: Address | null = null;
    CoBorrowerEmailAddress: string = "";
    CoBorrowerPhone: string = "";
    CoBorrowerEmployerName: string = "";
    CoBorrowerEmploymentHireDate: string | null = null;
    CoBorrowerIncome: number | null = null;
    Amount: number | null = null;
    Purpose: string = "";
    MortgagePayment: number | null = null;
    RentPayment: number | null = null;
    CreateTimestamp: string | null = null;
    UpdateTimestamp: string | null = null;
    Comments: LoanApplicationComment[] | null = [];
}
